### ISSUE TITLE
Add the opt-in show-attendees list on the event page

### ISSUE THREAD

**Issue by kharann** [dotkom/onlineweb-frontend#425]
**Is your feature request related to a problem? Please describe.**
Show attendees is missing from the event page! On the old page you could see choose if you want to be anon or be on a public attendee list

**Describe the solution you'd like**

Add this opt-in attendee list to the event details page!

**Comment by FluidSense** [comment]
This is currently a completely different page! `/event/:id/public-attendees` to display. Requires login.
Should perhaps be moved to a modal on event

**Comment by oleast** [comment]
Looks like the slug redirect takes precedence over the `/public-attendees` path :upside_down_face:

**Comment by FluidSense** [comment]
> Looks like the slug redirect takes precedence over the /public-attendees path 🙃
Yeah, this makes sense..! The redirect is done by Next _before_ it checks its own paths.
Could perhaps be fixed by using a ^ at the end of the URL, if the redirect supports regex

**Comment by Paalar** [comment]
Partially solved by #480 
Though it is missing the button to show attending for a single event.
In my opinion, I feel like it is weird to show attending for just one event and not all events. It also adds more complexity to DB models and code. I feel like you should either show for all events or none events, which is set in the profile.

**Comment by oleast** [comment]
> In my opinion, I feel like it is weird to show attending for just one event and not all events. It also adds more complexity to DB models and code. I feel like you should either show for all events or none events, which is set in the profile.

There were a lot of discussions about this when it was first implemented, and the current system can be made easy if it is implemented as first thought out. Since it is a consent it is nice to give the user granular controls, it is also implemented this way for `allow_pictures/photo_consent`, which has to let the user decide for each event.

How it is thought to work is that you have the global profile setting for `visible_as_attending_events`, and every time you regiter for an event that setting will be used as a default. Then you can change that default while registering the event (or after).

**Comment by Paalar** [comment]
I agree that `allow_pictures/photo_consent` should be on a per event basis, but I don't think that just because you can set `allow_pictures/photo_consent` per event, you should be able to set visibility per event. Their meaning and importance is quite different from each other.

I believe that most people that set themselves as visible only per event basis, want it to be visible on all events, but do not know of the setting in their profile.
